api/src/test/java/org/pmiops/workbench/api/UserMetricsControllerTest.java

```diff
@@ -155,6 +155,67 @@ public void setUp() {
 
   }
 
+  /**
+   * Note about this test vs other tests. This is intended to be run
+   * independently from other tests, therefore we're doing resource
+   * creation here instead of in setUp.
+   */
+  @Test
+  public void testGetUserRecentResourceFromRawBucket() {
+    List<UserRecentResource> userRecentResources = new ArrayList<>();
+    UserRecentResource resource1 = new UserRecentResource();
+    resource1.setNotebookName("gs://bucketFile/notebook.ipynb");
+    resource1.setCohort(null);
+    resource1.setLastAccessDate(new Timestamp(clock.millis()));
+    resource1.setUserId(USER_ID);
+    resource1.setWorkspaceId(WORKSPACE_1_ID);
+    userRecentResources.add(resource1);
+    when(userRecentResourceService.findAllResourcesByUser(USER_ID))
+        .thenReturn(userRecentResources);
+    userMetricsController = new UserMetricsController(
+        userProvider,
+        userRecentResourceService,
+        workspaceService,
+        fireCloudService,
+        clock);
+
+    RecentResourceResponse recentResources = userMetricsController
+        .getUserRecentResources().getBody();
+    assertNotNull(recentResources);
+    assertEquals(recentResources.get(0).getNotebook().getPath(), "gs://bucketFile/");
+    assertEquals(recentResources.get(0).getNotebook().getName(), "notebook.ipynb");
+  }
+
+  /**
+   * Note about this test vs other tests. This is intended to be run
+   * independently from other tests, therefore we're doing resource
```

Comment:
Wording/semantic nit: All (good) tests are meant to be run independently of other tests. I'm guessing the intent here was to express that the setup is different than the standard test setup in this file.

Reply:
Yes, that is true. I'd prefer that the test setup didn't do what it does, but I think that's a separate refactor (I'll likely take that on soon).

Reply:
Removed these comments and added a TODO before the setup. Will tackle that refactor separately.